returnn/training.py

```diff
@@ -252,6 +275,19 @@ def _relink(src, dst):
         os.link(src, dst)
 
     def run(self):
+        if self.pe_num_processes:
+            # Some useful debugging:
+            if "PE_HOSTFILE" in os.environ:
```

Comment:
Note: Also this is SGE specific. Not sure if this is a problem though. Maybe I just should extend the comment.